web/src/utils/getIcon.jsx

```diff
@@ -2,7 +2,7 @@ import spotify from "@/../static/imgs/spotify.png";
 import dlna from "@/../static/imgs/dlna.png";
 import bluetooth from "@/../static/imgs/bluetooth.png";
 import fmradio from "@/../static/imgs/fmradio.png";
-import shairport from "@/../static/imgs/shairport.png";
+import shairport from "@/../static/imgs/shairport.svg";
```

Comment:
Any reason this is svg instead of png?

Reply:
We use the icons in both large and small formats, svgs are better at not having jagged edges when you use CSS to scale them like that

However, I also grabbed a PNG because without one I'd have to change every other stream icon to SVG as well, and that's a larger lift than this issue deserved at this moment

I've decided to skip having two different image files and just keep it to PNGs